github/ipfs.yml

```diff
@@ -16,6 +16,7 @@ members:
     - olizilla
     - protocollabsit
     - Stebalien
+    - tinytb
```

Comment:
I would suggest we make you a member rather than admin since we already want to get this list shrinked down and don't want to make it worse.  With github-mgmt, the need for many to be admins goes down.

The more controlled way to have admin permissions is to add yourself to the github-mgmt-stewards team.

Reply:
In https://github.com/ipfs/github-mgmt/pull/71/commits/522a0129a2a92a2824f88da862cd322147f6d856 moved to `github-mgmt stewards` -- is that what you meant?

Reply:
Sorry @tinytb to not be clear. 

I think it's fine/good to have you part of w3dt-stewards so you have access to many of our repos.

I was reacting to you adding yourself as an "admin" to the organization.  We want to reduce the number of people have "admin" permission.  The main reason people need to be admins for the org is so they can approve adding other members, modify teams, etc.  The way to do that is through github-mgmt and folks on the github-mgmt-stewards team are who can merge changes in.  Basically the blessed way to be an "admin" is to join github-mgmt-stewards.